### PR TITLE
release v0.6.2

### PR DESCRIPTION
**Pull request by pekura**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,22 @@ The lerna-changelog tool detects changes based on PR labels and maps them to sec
     ...
   }
 -->
+## [v0.6.2] (2019-08-29)
+
+#### :rocket: Added
+* [#746](https://github.com/SAP/luigi/pull/746) Custom events for core client communication ([@maxmarkus](https://github.com/maxmarkus))
+* [#774](https://github.com/SAP/luigi/pull/774) Security alert fix ([@JohannesDoberer](https://github.com/JohannesDoberer))
```
Comment (jesusreal) [inline CHANGELOG.md]: Maybe we can remove this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,22 @@ The lerna-changelog tool detects changes based on PR labels and maps them to sec
...
+* [#756](https://github.com/SAP/luigi/pull/756) Add test attribute to navigation nodes ([@JohannesDoberer](https://github.com/JohannesDoberer))
+* [#752](https://github.com/SAP/luigi/pull/752) Mechanism for partial luigi config update ([@pekura](https://github.com/pekura))
```
Comment (jesusreal) [inline CHANGELOG.md]: Maybe here Luigi with uppercase

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,22 @@ The lerna-changelog tool detects changes based on PR labels and maps them to sec
...
+* [#745](https://github.com/SAP/luigi/pull/745) Extend iframe sandbox rules ([@JohannesDoberer](https://github.com/JohannesDoberer))
+
+#### :bug: Fixed
+* [#760](https://github.com/SAP/luigi/pull/760) Prevent luigi.auth.tokenIssued infinite loop ([@maxmarkus](https://github.com/maxmarkus))
+
+#### :memo: Documentation
+* [#717](https://github.com/SAP/luigi/pull/717) Add documentation for responsive app setup ([@jesusreal](https://github.com/jesusreal))
+* [#755](https://github.com/SAP/luigi/pull/755) Update codeowners file ([@alexandra-simeonova](https://github.com/alexandra-simeonova))
```
Comment (maxmarkus) [inline CHANGELOG.md]: guess we can remove this too

Comment (jesusreal) [inline CHANGELOG.md]: Maybe we can remove this line

File: CHANGELOG.md [hunk 4]
```diff
@@ -487,3 +503,4 @@ The lerna-changelog tool detects changes based on PR labels and maps them to sec
 [v0.5.4]: https://github.com/SAP/luigi/compare/v0.5.3...v0.5.4
 [v0.6.0]: https://github.com/SAP/luigi/compare/v0.5.4...v0.6.0
 [v0.6.1]: https://github.com/SAP/luigi/compare/v0.6.0...v0.6.1
+[v0.6.2]: https://github.com/SAP/luigi/compare/v0.6.2...v0.6.2
```
Comment (jesusreal) [inline CHANGELOG.md]:
```suggestion
[v0.6.2]: https://github.com/SAP/luigi/compare/v0.6.1...v0.6.2
```

File: CHANGELOG.md (hunk 2 above)
Comment (jesusreal) [inline CHANGELOG.md]:
```suggestion
* [#752](https://github.com/SAP/luigi/pull/752) Mechanism for partial Luigi config update ([@pekura](https://github.com/pekura))
```

File: CHANGELOG.md (hunk 3 above)
Comment (jesusreal) [reply CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md (hunk 1 above)
Comment (jesusreal) [reply CHANGELOG.md]:
```suggestion
```

**Pull request merged**